src/string.cr

```diff
@@ -497,7 +497,7 @@ class String
     if info.negative
       {% if max_negative %}
         return yield if info.value > {{max_negative}}
-        -info.value.to_{{method}}
+        (~info.value &+ 1).unsafe_as(Int64).to_{{method}}
```

Comment:
What does `~` mean? 🤔

Reply:
It's `Int#~`:
```cr
  def ~
    self ^ -1
  end
```
Source: [int.cr](https://github.com/crystal-lang/crystal/blob/master/src/int.cr#L77-L79)

Reply:
@vladfaust it's inverting bits to perform https://en.wikipedia.org/wiki/Two%27s_complement

Sample:
```
00000011 ( 3. info.value)   
11111100 (-4. ~info.value)
11111101 (-3. ~info.value &+ 1)
```

Reply:
Thanks, @z64, @bcardiff

Reply:
This method could use some API docs.